Update common-radio-control-calibration.rst for Mode 3 and 4 (#5939)
ArduPilot/ardupilot_wiki: common/source/docs/common-radio-control-calibration.rst

.. figure:: ../../../images/radio_setup_mode_1.png
   :target: ../_images/radio_setup_mode_1.png

There are two alternative configurations:

-  *Mode 3*: left stick controls pitch and roll, the right stick will
   control throttle and yaw.
-  *Mode 4*: left stick controls throttle and roll; the right stick will
   control pitch and yaw.

> As you can see, mode 3 is the opposite to mode 2 and mode 4 is the opposite to mode 1, giving complete right handed/left handed user options.

Reviewer: ```suggestion
Mode 3 is the opposite of Mode 2 and Mode 4 is the opposite of Mode 1, giving complete right handed/left handed user options.
```
